src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/experimental_detectron_prior_grid_generator.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <single_layer_tests/experimental_detectron_prior_grid_generator.hpp>
+#include <common_test_utils/data_utils.hpp>
+#include <functional_test_utils/ov_tensor_utils.hpp>
+
+namespace {
+
+const std::initializer_list<ov::test::subgraph::ExperimentalDetectronPriorGridGeneratorTestParam> params {
+    // flatten = true (output tensor is 2D)
+    {
+        {true, 0, 0, 4.0f, 4.0f},
+        ov::test::static_shapes_to_test_representation({{3, 4}, {1, 16, 4, 5}, {1, 3, 100, 200}})
+    },
+    {
+        {true, 3, 6, 64.0f, 64.0f},
+        ov::test::static_shapes_to_test_representation({{3, 4}, {1, 16, 100, 100}, {1, 3, 100, 200}})
+    },
```

Comment:
In case of TCs in which w and h were set to a non-zero value, I checked it currently failing. IMO, it would be better to re-enable this TC after these are commented like GPU plugin's layer test and making it pass later.

Reply:
done (in PR https://github.com/openvinotoolkit/openvino/pull/11632 )